includes/class-wcpdf-endpoint.php

```diff
@@ -177,6 +177,37 @@ public function get_document_link_access_type() {
 		return apply_filters( 'wpo_wcpdf_document_link_access_type', $access_type, $this );
 	}
 	
+	/**
+	 * Get document denied frontend redirect URL
+	 *
+	 * @return string
+	 */
+	public function get_document_denied_frontend_redirect_url() {
+		$redirect_url   = '';
+		$debug_settings = get_option( 'wpo_wcpdf_settings_debug', array() );
+		
+		if ( isset( $debug_settings['document_access_denied_redirect_page'] ) ) {
+			switch ( $debug_settings['document_access_denied_redirect_page'] ) {
+				case 'login_page':
+					$redirect_url = wp_sanitize_redirect( wp_login_url() );
+					break;
+				case 'myaccount_page':
+					$redirect_url = wp_sanitize_redirect( wp_login_url( wc_get_page_permalink( 'myaccount' ) ) );
```

Comment:
All the options are work correctly, except this one.

I think we need to remove the `wp_login_url()` function here, isn't?

Reply:
Are you logged in?

Reply:
@alexmigf I tested again in this is the result:

- Logged in: It redirects to the WP dashboard
- Guest user: It redirects to login page, not the My account page.